deploy/10-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-cost-exporter
+  namespace: monitoring
+  labels:
```

Comment:
app process is mandatory:

https://github.com/grupozap/k8s/blob/master/Documentation/howToTag.md#what-tags-should-i-use-in-my-apps

Reply:
Fixed! Thanksss